app/controllers/users_controller.rb

```diff
@@ -3,5 +3,8 @@ def index
     @users = User.all
   end
```

Comment:
Users controller `#index` action renders all users to its corresponding Views index page 🔥